src/NipValidator/NipValidator.cs

```diff
@@ -9,6 +9,8 @@ public class NipValidator
 
         public static bool IsValid(string nip)
         {
+            var sonarQubeTrap = 0;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'sonarQubeTrap' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1481)